tests/std/tests/GH_001914_Cached_position/test.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <ranges>
+
+#include <range_algorithm_support.hpp>
+using namespace std;
+
+template <class View>
+class test_cache : public ranges::_Cached_position<View, test_cache<View>> {
+public:
+    constexpr test_cache(View view) : _view(move(view)) {}

```

Comment:
Should include `<utility>` for `move()`. (`<range_algorithm_support.hpp>` includes it, but if tests are careful to include what they use, that allows the common headers to be more easily refactored in the future.)